src/ViewWidgets/LogDownloadController.cc

```diff
@@ -361,11 +365,13 @@ LogDownloadController::_logData(UASInterface* uas, uint32_t ofs, uint16_t id, ui
                 //-- Check for more
                 _receivedAllData();
             } else if (_chunkComplete()) {
+                qCDebug(LogDownloadLog) << "Chunk complete";
                 _downloadData->advanceChunk();
                 _requestLogData(_downloadData->ID,
                                 _downloadData->current_chunk*kChunkSize,
                                 _downloadData->chunk_table.size()*MAVLINK_MSG_LOG_DATA_FIELD_DATA_LEN);
             } else if (bin < _downloadData->chunk_table.size() - 1 && _downloadData->chunk_table.at(bin+1)) {
+                qCDebug(LogDownloadLog) << "Missing data";
```

Comment:
What is meant by this message? This scope is actually when a fragment of previously missing data has been successfully redownloaded.